templates/install_kb.ps1.erb

```diff
@@ -5,7 +5,7 @@ $all_error_codes = Get-Content -raw -Path $LibDir | ConvertFrom-StringData
 
 [void](Install-WindowsUpdate -KBArticleID "<%= @kb %>" -AcceptAll -IgnoreReboot)
 Start-Sleep 5
-$update = Get-WUHistory | ? KB -eq <%= @kb %> | Sort-Object Date -Descending | Select-Object -First 1
+$update = Get-WUHistory -Last 1 | ? KB -eq <%= @kb %>
```

Comment:
Recommend bumping this to an absurd number, to ensure the KB filter can actually match results. For example `-Last 5000`